src/drop-down-menu.jsx

```diff
@@ -65,6 +66,7 @@ let DropDownMenu = React.createClass({
 
   getStyles(){
     let zIndex = 5; // As AppBar
+    let disabled = this.props.disabled;
```

Comment:
@mjhasbach Understood. We're trying to update our files with ```const``` where appropriate and it looks like you happened to find a file we haven't touched yet. When accessing ```this.props``` you should always write
```javascript
// this.props is immutable within the context of this so we should treat it as such
const { style, children } = this.props; 
```
if you are going to expand ```this.props```.